META.coq

```diff
@@ -293,3 +293,288 @@ package "ide" (
   archive(native)  = "ide.cmxa"
 
 )
+
+package "plugins" (
+
+  description = "Coq built-in plugins"
+  version     = "8.7"
+
+  requires    = ""
+  directory   = "plugins"
```

Comment:
Are these two entries `requires` and `directory` mandatory?

Reply:
No entries are mandatory as far as I know. However, the `directory` directive in `plugins` is useful to tell ocamlfind to look for the subpackages in this directory. Otherwise it will look again from the Coq folder.

I will remove this empty `requires` for this specific package, but not for the other packages for uniformity between the plugins.

Reply:
Ok, thanks!